src/network/core/tcp_content_type.h

```diff
@@ -26,6 +26,7 @@ enum ContentType {
 	CONTENT_TYPE_GAME          = 9, ///< The content consists of a game script
 	CONTENT_TYPE_GAME_LIBRARY  = 10, ///< The content consists of a GS library
 	CONTENT_TYPE_END,               ///< Helper to mark the end of the types
+	CONTENT_TYPE_INVALID       = 0xFF, ///< Invalid/uninitialized content
```

Comment:
Code convention, and I am not sure I agree with it, but we do it everywhere, says `INVALID` is a prefix, not postfix. so `INVALID_CONTENT_TYPE`.
Not everyone got that memo btw, so there are some strays, but .. yeah .. you can see it with `CONTENT_ID` a bit lower too :)